ember-welcome-page/src/components/welcome-page.ts

```diff
@@ -0,0 +1,45 @@
+import { getOwner } from '@ember/application';
+/* @ts-expect-error: Cannot find module '@ember/version' or its corresponding type declarations. */
+import { VERSION } from '@ember/version';
+import Component from '@glimmer/component';
+import './welcome-page.css';
+
+function isLatestVersion(): boolean {
+  const stableRegex = /^\d+\.\d+\.\d+$/;
+
+  return !stableRegex.test(VERSION);
+}
+
+export default class WelcomePageComponent extends Component {
+  get rootURL(): string {
+    const config = getOwner(this).factoryFor('config:environment');
+
+    if (config) {
+      return (config.class as unknown as { rootURL: string }).rootURL;
+    }
+
+    return '/';
+  }
+
+  get emberVersion(): string {
+    if (isLatestVersion()) {
+      return 'current';
+    }
+
+    const [majorVersion, minorVersion] = VERSION.split('.');
+    const emberVersion = `${majorVersion}.${minorVersion}.0`;
+
+    return emberVersion;
+  }
```

Comment:
The template shows a data attribute called `data-ember-version`. I'm not sure if this data attribute is used in other projects.

To my knowledge, the data attribute is only used for this project's rendering tests and used incorrectly. What the rendering tests really want to assert is that the links to the Ember Guides will be correct. Such assertion can be done without the data attribute.

In short, we could delete the getter `emberVersion`, if we can verify that the data attribute `data-ember-version` is not used by external projects.